### PR TITLE
CR069: synchro with VDV736 / UmS v1.0 profile of SX

### PR DESCRIPTION
**Pull request by haeckerbaer**
See https://3.basecamp.com/3256016/buckets/9672657/vaults/3305260842.
CR069 in its original form (as presented and accepted in 2019, Berne), i.e., all changes and new structures from VDV736 / UmS documentation v1.0 are integrated into CEN version. Minimal differences still exist (will be summarized and uploaded to basecamp), but the goal is that a VDV736 XML delivery validates with the CEN XSD in the same way as with the UmS XSD (further testing will be done).

I updated the UmS changes according to our decisions on 2021-01-12 summarized here: https://3.basecamp.com/3256016/buckets/9672657/uploads/3305341112

Some enumeration differences have to be discussed before the merge:
https://3.basecamp.com/3256016/buckets/9672657/uploads/3371568599
AlertCauseEnumeration was updated from TPEG Pti19 to Pts38 independently in the CEN and UmS version. The problem is that a handful of values differ (Pts references is identical). I solved this by adding the UmS values with a comment "for compatibility" (I assumed that the CEN values were the correct ones). 
Please have a look at it here: 
https://github.com/SIRI-CEN/SIRI/blob/f9eb338081070237c4e59c6223bca2f59f0a487c/xsd/siri_model/siri_situationReasons-v2.0.xsd#L1265

### CODE REVIEW

**Comment by prirey** [comment]
Hi  I double checked the reason listings and would suggest the following changes:
In XSD
- define "UnknownReason" and "UndefinedReason" as deprecated: consistent to CEN external tables (values not documented and are substituted by unknown / undefinedAlertCause in PTS 38)
- breakdown / heavySnowfall (is lower case in external CEN tables, fits to English spelling)
- add awaitingOncomingVehicle (defined in CEN external tables, PTS 038 134), levelCrossingBlocked (defined in CEN external tables PTS 038 64), waitingForTransferPassengers(defined in CEN external tables PTS 038 130)
- add The in fireAtStation (consistent to CEN external tables)

In CEN external tables
- add the values awaitingApproach (Pts38_134), awaitingShuttle (Pts38_130)

@hennoit please let me know if you don't agree
I can update the CEN documentation part 5 and external tables.
@haeckerbaer I can update the UmS XSD/ documentation according after this pullrequest.

File: xsd/siri_model/siri_situationActions-v2.0.xsd [hunk 1]
```diff
@@ -467,4 +514,252 @@ Rail transport, Roads and road transport
...
+  </xsd:sequence>
+ </xsd:complexType>
+ <xsd:complexType name="DescriptionContentStructure">
+  <xsd:sequence>
+   <xsd:element name="DescriptionText" type="DefaultedTextStructure" maxOccurs="unbounded">
+    <xsd:annotation>
+     <xsd:documentation>Description of SITUATION. Should not repeat any strap line included in Summary.  (Unbounded since SIRI 2.0)</xsd:documentation>
+    </xsd:annotation>
+   </xsd:element>
+   <xsd:element name="DescriptionPriority" type="xsd:positiveInteger" minOccurs="0">
```
Comment (hennoit) [inline xsd/siri_model/siri_situationActions-v2.0.xsd:537]: Are the xxxPriority elements mandatory if there is more than one xxxContent - or can they be optional even then?

**Review by hennoit — CHANGES_REQUESTED** [review]
some small (textual) comments

**Comment by haeckerbaer** [comment]
Thanks @hennoit  for your detailed feedback! 
Will clean-up accordingly.
Cheers, A

**Review by Aurige — APPROVED** [review]
OK for me... with Henno's requested changes

**Review by prirey — CHANGES_REQUESTED** [review]
Hi Adrian, I found an error in SourceType, see correction attached.
image

`<xs:simpleType name="SourceTypeEnum">
  <xs:annotation>
   <xs:documentation>Type of sources from which SITUATION information may be derived.</xs:documentation>
  </xs:annotation>
  <xs:restriction base="xs:string">
   <xs:enumeration value="automobileClubPatrol">
    <xs:annotation>
     <xs:documentation>A patrol of an automobile club.</xs:documentation>
    </xs:annotation>
   </xs:enumeration>
   <xs:enumeration value="cameraObservation">
    <xs:annotation>
     <xs:documentation>A camera observation (either still or video camera).</xs:documentation>
    </xs:annotation>
   </xs:enumeration>
   <xs:enumeration value="freightVehicleOperator">
    <xs:annotation>
     <xs:documentation>An operator of freight vehicles.</xs:documentation>
    </xs:annotation>
   </xs:enumeration>
   <xs:enumeration value="inductionLoopMonitoringStation">
    <xs:annotation>
     <xs:documentation>A station dedicated to the monitoring of the road network by processing inductive loop information.</xs:documentation>
    </xs:annotation>
   </xs:enumeration>
   <xs:enumeration value="infraredMonitoringStation">
    <xs:annotation>
     <xs:documentation>A station dedicated to the monitoring of the road network by processing infrared image information.</xs:documentation>
    </xs:annotation>
   </xs:enumeration>
   <xs:enumeration value="microwavedMonitoringStation">
    <xs:annotation>
     <xs:documentation>A station dedicated to the monitoring of the road network by processing microwave information.</xs:documentation>
    </xs:annotation>
   </xs:enumeration>
   <xs:enumeration value="mobileTelephoneCaller">
    <xs:annotation>
     <xs:documentation>A caller using a mobile telephone (who may or may not be on the road network).</xs:documentation>
    </xs:annotation>
   </xs:enumeration>
   <xs:enumeration value="nonPoliceEmergencyServicesPatrol">
    <xs:annotation>
     <xs:documentation>Emergency service patrols other than police.</xs:documentation>
    </xs:annotation>
   </xs:enumeration>
   <xs:enumeration value="otherInformation">
    <xs:annotation>
     <xs:documentation>Other sources of information.</xs:documentation>
    </xs:annotation>
   </xs:enumeration>
   <xs:enumeration value="otherOfficialVehicle">
    <xs:annotation>
     <xs:documentation>Personnel from a vehicle belonging to the road operator or authority or any emergency service, including authorised breakdown service organisations.</xs:documentation>
    </xs:annotation>
   </xs:enumeration>
   <xs:enumeration value="policePatrol">
    <xs:annotation>
     <xs:documentation>A police patrol.</xs:documentation>
    </xs:annotation>
   </xs:enumeration>
   <xs:enumeration value="privateBreakdownService">
    <xs:annotation>
     <xs:documentation>A private breakdown service.</xs:documentation>
    </xs:annotation>
   </xs:enumeration>
   <xs:enumeration value="publicAndPrivateUtilities">
    <xs:annotation>
     <xs:documentation>A utility organisation, either public or private.</xs:documentation>
    </xs:annotation>
   </xs:enumeration>
   <xs:enumeration value="registeredMobileObserver">
    <xs:annotation>
     <xs:documentation>A motorist who is an officially registered observer.</xs:documentation>
    </xs:annotation>
   </xs:enumeration>
   <xs:enumeration value="roadAuthorities">
    <xs:annotation>
     <xs:documentation>A road authority.</xs:documentation>
    </xs:annotation>
   </xs:enumeration>
   <xs:enumeration value="roadOperatorPatrol">
    <xs:annotation>
     <xs:documentation>A patrol of the road operator or authority.</xs:documentation>
    </xs:annotation>
   </xs:enumeration>
   <xs:enumeration value="roadsideTelephoneCaller">
    <xs:annotation>
     <xs:documentation>A caller who is using an emergency roadside telephone.</xs:documentation>
    </xs:annotation>
   </xs:enumeration>
   <xs:enumeration value="spotterAircraft">
    <xs:annotation>
     <xs:documentation>A spotter aircraft of an organisation specifically assigned to the monitoring of the traffic network.</xs:documentation>
    </xs:annotation>
   </xs:enumeration>
   <xs:enumeration value="trafficMonitoringStation">
    <xs:annotation>
     <xs:documentation>A station, usually automatic, dedicated to the monitoring of the road network.</xs:documentation>
    </xs:annotation>
   </xs:enumeration>
   <xs:enumeration value="transitOperator">
    <xs:annotation>
     <xs:documentation>An operator of a transit service, e.g. bus link operator.</xs:documentation>
    </xs:annotation>
   </xs:enumeration>
   <xs:enumeration value="vehicleProbeMeasurement">
    <xs:annotation>
     <xs:documentation>A specially equipped vehicle used to provide measurements.</xs:documentation>
    </xs:annotation>
   </xs:enumeration>
   <xs:enumeration value="videoProcessingMonitoringStation">
    <xs:annotation>
     <xs:documentation>A station dedicated to the monitoring of the road network by processing video image information.</xs:documentation>
    </xs:annotation>
   </xs:enumeration>
  </xs:restriction>
 </xs:simpleType>`

**Comment by prirey** [comment]
Hi Adrian
In ServiceConditionEnumeration, I found in CR 15 External Tables that the following PTI values are deprecated. Can you fix the XSD?
altered
cancelled
delayed
noService
disrupted
additionalService
replacementService
undefinedServiceInformation
Thanks <3

**Comment by prirey** [comment]
Hi Adrian
I found a further inconsistency 
in siri_situation-v2.0xsd the QualityEnumeration is defined
image
in siri_journey_support-v2.0.xsd the QualityIndexEnumeration is defined
image
The values of both enumerations are identical in the xsd. However, there are two corrections I suggest:

- set default in the element PtSituationElement.QualityIndex to be consistent with the use of this enumeration in ConfidenceLevel (siri_monitoredVehicleJourney-v2.0.xsd). 

I don't know whether we want also to set this default in siri_journey-v2.0.xsd. 
image

image

- Delete the redundant QualityEnumeration.

File: xsd/siri_model/siri_situationActions-v2.0.xsd (hunk 1 above)
Comment (prirey) [reply xsd/siri_model/siri_situationActions-v2.0.xsd:537]: In the UmS group documentation it seems to be intentially optional. The UmS group documented that when priority is left out, end systems can decide how to sort/prioritise the text contents on its own. This seems reasonable for me.

**Pull request merged**